### PR TITLE
Fix datetime converter errors, speed up build time

### PR DESCRIPTION
**Pull request by Schwusch**
I made some mistakes naming the date serializer functions, which results in [broken packages](https://pub.dev/packages/aws_dynamodb_api/versions/0.1.0/score)(all packages with date serialization basically).
This fixes that problem hopefully, and **most important of all** it reduced the code generation time from ~80 minutes to ~12 on my machine with 8 cores! :tada:

### CODE REVIEW

File: shared_aws_api/lib/src/protocol/shared.dart [hunk 1]
```diff
@@ -8,15 +8,20 @@ import 'package:xml/xml.dart';
 final rfc822Formatter = DateFormat("EEE, dd MMM yyyy HH:mm:ss 'Z'", 'en_US');
 final iso8601Formatter = DateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", 'en_US');
 
-DateTime rfc822fromJson(String date) => rfc822Formatter.parse(date);
+DateTime rfc822FromJson(String date) =>
```
Comment (isoos) [inline shared_aws_api/lib/src/protocol/shared.dart:11]: This is technically a breaking change. Could you please also update the version and Changelog highlighting it?

Comment (Schwusch) [reply shared_aws_api/lib/src/protocol/shared.dart:11]: Considering it was broken before, should this perhaps be a patch increment? I'll bump it to 0.2.1..

Comment (isoos) [reply shared_aws_api/lib/src/protocol/shared.dart:11]: > Considering it was broken before, should this perhaps be a patch increment?

👀 :D

**Comment by isoos** [comment]
And the packages... de they need to be re-generated and re-published?

**Comment by Schwusch** [comment]
> re-generated and re-published?

They do, but it hopefully won't take all night with this change 😄  Depending on the number of cores of course.
Perhaps they should be depending on shared `^0.2.1`?

**Pull request merged**